Sources/ArgumentParser/Parsable Properties/Errors.swift

```diff
@@ -20,7 +20,7 @@ import MSVCRT
 /// An error type that is presented to the user as an error with parsing their
 /// command-line input.
 public struct ValidationError: Error, CustomStringConvertible {
-  var message: String
+  public internal(set) var message: String
```

Comment:
This is a good idea, but since it's an API change, let's handle it in a separate PR.

Reply:
This was a hangover from the previous protocol based design. I will open a separate PR.

Reply:
@natecook1000 This change is no in a separate PR #119